charts/stable/mosquitto/Chart.yaml

```
@@ -3,7 +3,7 @@ apiVersion: v2
 appVersion: 2.0.14
 description: Eclipse Mosquitto - An open source MQTT broker
 name: mosquitto
-version: 4.5.0
+version: 4.6.0
```

```suggestion
version: 4.5.1
```

Also don't forget to update the changelog annotation.

---

Both done